firebase_admin/_user_mgt.py

```diff
@@ -65,6 +71,16 @@ def last_sign_in_timestamp(self):
         """
         return self._last_sign_in_timestamp
 
+    @property
+    def last_refresh_timestamp(self):
+        """The time at which the user was last active (ID token refreshed).
+
+        Returns:
+          integer: Milliseconds since epoch timestamp, or None if the user was
```

Comment:
Does it literally return the string "None?"  The cap makes me thinks so . . . and if so this should be backticked probably.

Reply:
No, it returns the python `None` value (roughly corresponding to null in other languages). But backticks seem appropriate regardless (done).